mmpose/datasets/datasets/hand/interhand3d_dataset.py

```diff
@@ -258,6 +262,9 @@ def _get_db(self):
                 center, scale = self._xywh2cs(*bbox, 1.0)
                 abs_depth = rootnet_ann_data['abs_depth']
             rel_root_depth = joint_cam[41, 2] - joint_cam[20, 2]
```

Comment:
Explain `20` and `41`

Reply:
20 and 41 is the wrist(root) joint of right and left hand respectively .

Reply:
Add these note in line #264

Reply:
fixed